### PR TITLE
[WIP] Doc mod

### PR DESCRIPTION
**Pull request by matthiasbeyer**
Add high-level description what each module does.

---

The following modules are planned:

- [ ] bookmarks
- [ ] borrow-tracker
- [ ] calendar (via external tools, E.G.: vdirsyncer)
- [ ] contacts (via external tools, E.G.: vdirsyncer)
- [ ] counter
- [ ] cue cards
- [ ] dairy
- [ ] images (via extenal tools, E.G.: git-annex)
- [ ] ledger (via external tools, E.G.: ledger-cli)
- [ ] mails (via extenal tools, E.G.: mutt, offlineimap, msmtp)
- [ ] movies (via extenal tools, E.G.: git-annex)
- [ ] music (via extenal tools, E.G.: git-annex)
- [ ] news (via external tools, E.G.: newsbeuter)
- [ ] notes
- [ ] password (via external tools, E.G.: password-store)
- [ ] personal library
- [ ] todo (via external tools, E.G.: taskwarrior)
- [ ] weather
- [ ] wiki

and also some utility modules:

- [ ] link helper
- [ ] tag helper
- [ ] git helper
- [ ] content-to-website (markup converter for viewing things in the browser)
- [ ] store-integrity check (find dead links, etc)

### CODE REVIEW

File: doc/src/06000-modules.md [hunk 1]
```diff
@@ -18,3 +18,5 @@ By definition, the Library depends on the libstore (@sec:libstore).
 A module MUST use the runtime library to implement the commandline
 interface as defined in @sec:librt.
 
+The following sections describe the functionality of the modules.
```
Comment (neithernut) [inline doc/src/06000-modules.md]: Maybe we should differentiate a bit between modules which are part of this project and third party modules based on the imag libraries. I would propose calling them "build-in modules" or "modules shipped with the imag core".

Comment (TheNeikos) [inline doc/src/06000-modules.md]: :+1:

**Comment by matthiasbeyer** [comment]
I think we should not differ. It is implicit that these are internal modules. For external things we can have a dedicated section later...

File: doc/src/06000-modules.md (hunk 1 above)
Comment (matthiasbeyer) [inline doc/src/06000-modules.md]: I think we should not differ. It is implicit that these are internal modules. For external things we can have a dedicated section later...

**Comment by neithernut** [comment]
What do you think of merging `06000-modules.md` with this PR rather soonish and adding the module specific sections afterwards? Maybe in the PRs introducing the code belonging to them?

**Comment by matthiasbeyer** [comment]
Yep, sounds good, though I'd like to open another PR for this, okay?

**Comment by neithernut** [comment]
I have no objections.

**Comment by matthiasbeyer** [comment]
Just to understand you correctly: The changes in `06000` is exactly one line. I'd rather close this PR and let the PRs for the modules include the module descriptions.

**Comment by neithernut** [comment]
Yes, I'm fine with that.

**Comment by matthiasbeyer** [comment]
On hold until the commits are cherry-picked into other PRs.

**Pull request closed without merging**